src/database.d.ts

```diff
@@ -1272,7 +1272,6 @@ export namespace admin.database {
      * ```
      */
     root: admin.database.Reference;
-    path: string;
```

Comment:
Not sure if we should make this change, but this doesn't exist in the Web types.

Reply:
If `Reference.path` is truly undefined let's remove it. Otherwise, let's call it deprecated for now and remove from a future major version.

Reply:
We can make it deprecated. We removed in a major release on Web as well.

Reply:
It makes no sense to remove this. Why require developers to individually create helper functions to fetch the string path from every reference instead of keeping this useful variable?

Every time I deploy my functions I'm greeted by a wall of "path is depreciated" messages... Why should I need to make a function called getPath(reference) that is prone to errors and needed to be referenced throughout my entire codebase when this does exactly what it needs to do?

Consider continuing support for this since the justification to depreciate it is so flimsy.